_component_design/navigation.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: Navigation
-layout: default
-category: Layout
+parent: null
+layout: component
```

Comment:
For now, this layout will be all right. Eventually, this will change when we get a family page template generated with the new site design.